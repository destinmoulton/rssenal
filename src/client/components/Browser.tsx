import debug from "debug";
import * as React from "react";

import LeftPaneContainer from "../containers/LeftPane/LeftPaneContainer";
import MessagesContainer from "../containers/MessagesContainer";
import RightPaneContainer from "../containers/RightPane/RightPaneContainer";

const log = debug("rssenal:Browser");

const Browser: React.SFC = () => {
    log("Browser()");
    return (
        <div>
            <div className="rss-body-left-container">
                <LeftPaneContainer />
            </div>
            <div className="rss-body-right-container">
                <RightPaneContainer />
            </div>
            <MessagesContainer />
        </div>
    );
};

export default Browser;
