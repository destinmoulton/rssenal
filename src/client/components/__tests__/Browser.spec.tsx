import * as React from "react";
import { shallow } from "enzyme";

declare global {
    namespace NodeJS {
        interface Global {
            Headers: () => void;
        }
    }
}

global.Headers = jest.fn().mockImplementation(() => {
    return { append: () => {} };
});
import Browser from "../Browser";

describe("<Browser />", () => {
    it("renders a snapshot", () => {
        const wrapper = shallow(<Browser />);

        expect(wrapper).toMatchSnapshot();
    });
});
