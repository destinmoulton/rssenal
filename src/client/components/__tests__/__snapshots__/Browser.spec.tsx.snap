// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Browser /> renders a snapshot 1`] = `
<div>
  <div
    className="rss-body-left-container"
  >
    <Connect(LeftPaneComponent) />
  </div>
  <div
    className="rss-body-right-container"
  >
    <Connect(RightPaneComponent) />
  </div>
  <Connect(MessagesComponent) />
</div>
`;
